### PR TITLE
Add Styleguidist for docs

### PR DESCRIPTION
**Pull request by gcandal**
| - | - |
| --- | --- |
| Decisions | Uses Styleguidist for auto-generating docs based on comments. |

### CODE REVIEW

**Comment by joamag** [comment]
@gcandal migrate the package.json according to https://github.com/ripe-tech/ripe-pulse/commit/2ce0b475e2b8ec522bcb80c275c63a5f71dd5f3d

**Comment by gcandal** [comment]
@joamag right now this doesn't work because of `Uncaught ReferenceError: regeneratorRuntime is not defined`.

This can be fixed by [adding](https://github.com/hivesolutions/uxf_webpack/blob/master/config/webpack.config.js#L55) a plugin to `babel-loader`'s `options`:

```
                           plugins: [
                                [
                                    "@babel/plugin-transform-runtime",
                                    {
                                        regenerator: true
                                    }
                                ]
                            ]
```

File: package.json [hunk 1]
```diff
@@ -76,7 +78,8 @@
         "node-fetch": "^2.6.0",
         "svgo": "^1.3.2",
         "url-loader": "^4.1.0",
-        "uxf-webpack": "^0.5.5"
+        "uxf-webpack": "^0.5.5",
```
Comment (joamag) [inline package.json:81]: let's remove this and add all the stuff

Comment (joamag) [reply package.json:81]: vue, babel, webpack, etc

**Review by joamag — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gcandal) [reply package.json:81]: Addressed in https://github.com/ripe-tech/ripe-sdk-components-vue/pull/22

**Review by joamag — APPROVED** [review]

**Pull request merged**